src/services/transaction-service.js

```diff
@@ -0,0 +1,11 @@
+import { StorageService } from "./storage";
+
+export class TransactionListService {
+  static get() {
+    return StorageService.get("transactionList");
+  }
+
+  static set(value) {
+    return StorageService.set("transactionList", value);
+  }
```

Comment:
I would like to see a method like `add` -- that will add a single transaction -- and you should use it in your code